src/cli/cmd/login/mod.rs

```diff
@@ -126,24 +182,54 @@ impl LoginSubcommand {
                 .ok_or_else(|| color_eyre::eyre::eyre!("cache_addr had no host"))?,
         );
 
+        let mut succeeded = false;
+
+        if let Some(mut uds) = dnee_uds {
+            tracing::info!("trying to update netrc via DNEE");
+
+            let add_req = NetrcTokenAddRequest {
+                token: token.clone(),
+                netrc_lines: netrc_contents.clone(),
+            };
+            let add_req_json = serde_json::to_string(&add_req)?;
+            let request = http::request::Builder::new()
+                .uri("http://dnee.socket/enroll-netrc-token")
+                .method(Method::POST)
+                .header("Content-Type", "application/json")
+                .body(Body::from(add_req_json))?;
+            let response = uds.send_request(request).await?;
+
+            let body = response.into_body();
+            let bytes = body.collect().await.unwrap_or_default().to_bytes();
+            let text: String = String::from_utf8_lossy(&bytes).into();
+
+            tracing::info!("sent the add request: {:?}", text);
+
+            succeeded = true;
+        }
+
+        if !succeeded {
```

Comment:
Would be nice to have some logging here, like "unable to connect to Determinate Nix socket, falling back to file-based approach" or something